tests/e2e/cucumber/features/smoke/sort.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Sort files/folders
+  As a user
+  I want to sort files/folders
+  So that I can make the file/folder list more clear
```

Comment:
According to https://github.com/owncloud/web/issues/10259#issuecomment-2009826676, we wont be adding sorting e2e tests